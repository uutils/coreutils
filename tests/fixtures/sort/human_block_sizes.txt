14M
20M
20P
11M
981T
16M
18M
19M
844K
981K
13M
K