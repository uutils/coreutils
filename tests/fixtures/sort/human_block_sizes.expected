K
844K
981K
11M
13M
14M
16M
18M
19M
20M
981T
20P
